### DISCUSSION TITLE
Source code

### DISCUSSION THREAD

**Discussion by ChandruMuthuswamy** [community/community#132212] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

we can get source code for projects

**Comment by VyvaHart** [comment, marked as answer]
Hi @ChandruMuthuswamy 

If it is a question there are two main categories of source code:

- Open-source code: This code is freely available for anyone to use, modify, and share
- Proprietary code: This code is owned by someone or some company, and you typically need permission to use it

good luck!

**Reply by ChandruMuthuswamy** [reply]
Thank you
